### ISSUE TITLE
Swiper: adding new cards programatically

### ISSUE THREAD

**Issue by simplenotezy** [appinioGmbH/flutter_packages#77]
I guess a common use case would be to load in more cards to the back, based on some condition (e.g. when less than 3 cards left), but I am not sure if the library supports it, or how to go about it.

Any suggestions?

**Comment by simplenotezy** [comment]
I see now that the new version uses ItemBuilder which is pretty cool, so the count can simply be incremented (e.g. by appending to `cards` array)

**Comment by simplenotezy** [comment]
Yes, I just saw that @khanmujeeb687 - super great additions. I was a bit confused first when I tried to align our own fork with current changes - but positively surprised to see this change! 😊

**Comment by bilalhamud** [comment]
Solved with version 2.0.0

**Comment by mafzaldev** [comment]
@bilalhamud Hi Bilal, is there any way to remove all the elements, at the _onEnd and add new elements on the go?
